core/run_ctest/main.py

```diff
@@ -23,16 +23,27 @@ def main():
             print(">>>>[ctest_core] input conf file: {}".format(conf_file_path))
             test_input = extract_conf_diff(conf_file_path)
             test_conf_file(conf_file_path, test_input)
+    elif run_mode == "run_single_ctest":
+        for conf_file_path in sorted(glob.glob(os.path.join(p_input["conf_file_dir"], "*"))):
+            print(">>>>[ctest_core] input conf file: {}".format(conf_file_path))
+            test_input = extract_conf_diff(conf_file_path)
+            test_conf_file(conf_file_path, test_input, argv[1])
```

Comment:
@Alex-Lian  do you want to run a single ctest on all configuration files in `conf_file_dir`, or a single ctest on any (parameter, value) you specify?

Reply:
The former one, in that case, we can test different groups of parameters at once. BTW, in most cases, we can remain only one file in the folder which records the parameter changes.